extensions/wikia/Recirculation/js/trackers/liftigniter.js

```diff
@@ -17,3 +17,4 @@ $p("init", "l9ehhrb6mtv75bp2", {
     }
 });
 $p("send", "pageview");
+$p("setRequestFields", ["rank", "thumbnail", "title", "url", "presented_by", "author"]);
```

Comment:
It was working in the helper, but it doesn't really belong there. A helper is created for each experiment. We're planning to have 3 experiments on the page at once (rail, impact footer, footer), so `setRequestFields` will get called three times. 

I see `trackers/liftigniter.js` as more of an initialization step rather than a tracking step. It inserts the liftigniter sdk onto the page, configures our usage of it, and then tracks the pageview. Tracking the pageview is probably better done somewhere else, but `setRequestFields` feels like a setup step to me.